lib/ProductOpener/Ingredients.pm

```diff
@@ -2292,9 +2285,9 @@ sub parse_ingredients_text ($product_ref) {
 
 					# Try to remove ingredients processing "cooked rice" -> "rice"
 					if (defined $ingredients_processing_regexps{$ingredients_lc}) {
-						($processing, $ingredient, $ingredient_id, $ingredient_recognized)
-							= parse_processing_from_ingredient(\%ingredients_processing_regexps,
-							$ingredients_lc, $processing, $ingredient, $ingredient_id, $and, $ingredient_recognized);
+						(my $new_processings_ref, $ingredient, $ingredient_id, $ingredient_recognized)
```

Comment:
Not necessarily new for new_processings_ref
"If it does not result in known ingredient, then it returns the same but unchanged." written in POD of the function.